pkg/resource/resource.go

```diff
@@ -105,7 +105,7 @@ func UpdateResourceVersion(req *unstructured.Unstructured, found *unstructured.U
 	if kind == "Service" {
 		clusterIP, fnd, err := unstructured.NestedString(found.Object, "spec", "clusterIP")
 		if err != nil || !fnd {
-			return fmt.Errorf("error or not found: %w", err)
+			return fmt.Errorf("error or clusterIP found: %w", err)
```

Comment:
should it be:
"error or clusterIP not found"?

Reply:
Good catch, thanks - fixed.